src/crosshook-native/src/styles/library.css

```
@@ -817,3 +823,422 @@
   color: var(--crosshook-color-text-muted);
   overflow-wrap: break-word;
 }
+
+/* ── Hero Detail (in-shell library detail mode) ───────────────────────── */
+
+.crosshook-hero-detail {
+  --crosshook-hero-detail-hero-height: min(240px, 28vh);
+  display: flex;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)